### ISSUE TITLE
"messenger is currently unavailable" keeps flashing every minute

### ISSUE THREAD

**Issue by mastef** [rsms/fb-mac-messenger#446]
Since recent versions this red notice keeps popping up quickly every 30-60 seconds. It's very distracting when you have the messenger visible : 

![messenger-error](https://user-images.githubusercontent.com/762297/63242973-58ede200-c283-11e9-9ff1-a577e45380aa.gif)

Any idea what it could be?

**Comment by dylankolson** [comment]
thought this was just my crappy internet - saw this too

**Comment by vdovhanych** [comment]
This happens, whenever the window is visible but not active, happens to me too. No idea to what could be the culprit.

**Comment by Lanithane** [comment]
This happens on messenger.com as well. I don't know what the cause is, but just commenting to give more visibility on the issue.

**Comment by mastef** [comment]
Then this sounds like an issue with the messenger. Any idea how to report bugs to the fb messenger team effectively?

**Comment by cdeutsch** [comment]
Hiding the class `._p0g.error` would make things less annoying.

This project looks a bit dead though based on outstanding PRs and last release date. 😬

![image](https://user-images.githubusercontent.com/404765/66226020-780fca00-e69f-11e9-8563-947c1518b588.png)

**Comment by mastef** [comment]
@cdeutsch you are right, but since react classes are auto-generated that class name may change any time soon
